src/test/java/com/google/javascript/clutz/empty_type_args.d.ts

```diff
@@ -0,0 +1,31 @@
+declare namespace ಠ_ಠ.clutz_internal.empty_type_args {
+  interface ITemplated < T > {
+  }
+}
+declare module 'goog:empty_type_args.ITemplated' {
+  import alias = ಠ_ಠ.clutz_internal.empty_type_args.ITemplated;
+  export default alias;
+}
+declare namespace ಠ_ಠ.clutz_internal.empty_type_args {
+  class NoMoreTemplateArgs implements ITemplated < number > {
```

Comment:
I don't understand which line here was absent before the fix. Was there no `<number>` here?